pom.xml

```diff
@@ -164,6 +164,23 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+          <version>3.1.12</version>
+          <configuration>
+            <excludeFilterFile>src/main/config/spotbugs-exclude-filter.xml</excludeFilterFile>
+          </configuration>
+          <executions>
+            <execution>
+              <id>quality-metric-spotbugs</id>
+              <phase>verify</phase>
```

Comment:
Is this working for you?
`mvn verify` isn't running spotbugs check goal for me

Reply:
Not working. It seems to be a problem with tag <execution>.

Reply:
Solved, tag plugins wasn't in the pom